src/components/calculator/PointsDisplay.tsx

```diff
@@ -29,49 +35,296 @@ function displayPercent(points: number): string {
 
 export function ExplanationCard(props: {
   points: number
+  lowerBound: number
+  upperBound: number
   repeatedEvent: boolean
   riskBudget: number
   riskBudgetSetter: (newValue: number) => void
+  data: CalculatorData
 }): React.ReactElement {
   const points = props.points
 
-  const activeRiskLevel = howRisky(points, props.riskBudget)
+  const [displayCalculationExplanation, setCalculationExplanation] = useState(
```

Comment:
showCalculatorExplanation, setShowCalculatorExplanation

Its really confusing if the setter uses a different name.